### ISSUE TITLE
KAT search URI no longer valid, causing 404 error

### ISSUE THREAD

**Issue by forkless** [SickRage/SickRage#1353]
Branch/Commit: master/6e3ebea4d634d3e5894ae9b85eabffe4284be0ca
OS: Linux
What you did: Enable a search with KAT torrent search enabled
What happened: Log dumps a 404 error with the constructed URI
What you expected: A search request that does not generate a 404
Logs:
```
2016-04-06 07:27:42 INFO     SEARCHQUEUE-MANUAL-304196 :: [KickAssTorrents] :: Request failed: 404 Client Error: Not Found for url: https://kat.cr/usearch/?q=Some+Show+S01E01&field=seeders&rss=1&category=tv&sorder=desc
2016-04-06 07:27:42 INFO     SEARCHQUEUE-MANUAL-304196 :: [KickAssTorrents] :: Performing episode search for Some Show
```

**Comment by SickRage2** [comment]
Thanks for the issue report! Before a real human comes by, please make sure your report has all the below criteria checked

- [ ] Include basic information: Branch/Commit, OS, What you did, What happened, What you expected
- [ ] Enable debug logging (be sure to disable after the bug is fixed)
- [x] Post debug logs, either inline (for smaller logs) or using gist
- [x] Please wrap inline logs inside \`\`\` and \`\`\` for readability

Please make sure you also read [how to create an issue](https://github.com/SickRage/sickrage-issues/blob/master/README.md) and followed all of the steps. 

The title should describe your issue. Having "SR not working" or "I get this bug" for 100 issues, isn't really helpful. We will close issues if there isn't enough information.

Sometimes the devs may seem like grunts and respond with short answers. This isn't (always) because the dev hates you, but because he's on mobile or busy fixing bugs. If something isn't clear, please let us know, and this bot may get updated to automatically answer you.

Thanks!

**Comment by miigotu** [comment]
its not a 404: https://kat.cr/usearch/?q=The+100+S01E01&field=seeders&rss=1&category=tv&sorder=desc

Did you change that to "Some+Show+S01E01" ?

**Comment by forkless** [comment]
Yes, the error however remains the same with a proper show.

**Comment by OmgImAlexis** [comment]
It's normal for torrent providers to display blank pages for RSS feeds when it doesn't match an item.

If you search for something that'll actually give a result you get the expected page.
Also if this was causing issues I'd have noticed as I just blacklogged 1,400+ episodes via KAT about 2 hours ago and none of them errored out.
![screen shot 2016-04-06 at 3 57 02 pm](https://cloud.githubusercontent.com/assets/6525926/14307707/407fe44e-fc10-11e5-8afc-6b03fb4f5bea.png)

**Comment by miigotu** [comment]
@forkless more likely kat was blocked by your ISP or something. KAT and TPB are the most commonly blocked of all internet sites (not just torrent sites, ALL sites). Thats what the custom url field is for

**Comment by forkless** [comment]
I see how that makes sense, the output in the log however does not reflect that very well. Now it suggests the search is broken. Other torrent providers I use just leave less noise in the log.

**Comment by OmgImAlexis** [comment]
@forkless if you can access https://kat.cr/usearch/?q=The+100+S01E01&field=seeders&rss=1&category=tv&sorder=desc in your browser and Sickrage is installed on the same machine then it's another issue.

**Comment by forkless** [comment]
Like I said, you are right it didn't generate an RSS feed because there is no hit on the search. However in contrast to the other torrent providers I use in SR this one is dumping additional info that suggests something errored out instead of not yielding a result.

**Comment by OmgImAlexis** [comment]
If you mean by "It is asking to get reported." it has a submit error button then use it.
